static/app/components/quickTrace/utils.tsx

```diff
@@ -82,14 +86,15 @@ export function generateSingleErrorTarget(
   event: TraceError | TracePerformanceIssue,
   organization: OrganizationSummary,
   location: Location,
-  destination: ErrorDestination
+  destination: ErrorDestination,
```

Comment:
Would it be possible to convert these function params to an object? 5 positional params seems like a bit much especially if some are optional

Reply:
@malwilley Possible but with TS checks doesn't seems like a big deal